src/BuildHistory.tsx

```diff
@@ -115,10 +122,13 @@ const BuildHistory = () => {
                                         color: "gray",
                                         fontStyle: "italic"
                                     }}>
-                                        Last Updated: {entry.updatedAt}
+                                        Last Updated: {entry.updatedAt.toISOString ? entry.updatedAt.toISOString() : entry.updatedAt}
```

Comment:
It might be simpler to wrap this in a template string (e.g. ``Last Updated; ${entry.updatedAt}`` to force serialization (instead of programmatically calling `toISOString` if it is a true `Date` object (i.e. when it hasn't been deserialized from serialized JSON from `LocalStorage`)